### PR TITLE
Hub page layout

### PR DESCRIPTION
**Pull request by BillWagner**
This PR includes the work we (me, @mairaw @mcleblanc) did in our working session Thursday:

- Move the "build any kind of app" to below the languages and above the reference section.
- Tighten the language on the highlights section.
- Re-work the conceptual section. 

On *Re-work the conceptual section*: 

From our notes, I looked at page views for key areas of .NET Core, .NET Standard, and .NET Framework areas. That suggested the three left-most cards in the conceptual section. 

Open questions:

- Where should a "See more" link direct to in "Develop .NET Apps"?
- All four cards include ".NET". Good or bad repetition?
- Are important links missing?

This is ready for a complete general review. [Internal review link](https://review.docs.microsoft.com/en-us/dotnet/index?branch=pr-en-us-15932)

### CODE REVIEW

File: docs/index.yml [hunk 1]
```diff
@@ -28,25 +28,138 @@ highlightedContent:
...
+          text: Deploying .NET Core apps
+        - url: core/docker/build-container.md
+          itemType: tutorial
+          text: Containerize .NET Core apps with Docker
+        - url: core/tutorials/index.md
+          itemType: tutorial
+          text: Tutorials
+        - url: core/porting/index.md
+          itemType: how-to-guide
+          text: Port  apps from .NET Framework
```
Comment (pkulikov) [inline docs/index.yml]:
```suggestion
          text: Port apps from .NET Framework
```

File: docs/index.yml [hunk 2]
```diff
@@ -28,25 +28,138 @@ highlightedContent:
...
+          text: Unit testing with .NET Core apps
+        - url: core/porting/index.md
+          itemType: how-to-guide
+          text: Porting from .NET Framework to .NET Core
+        - url: desktop-wpf/overview/index.md
+          itemType: overview
+          text: Create .NET Core desktop apps
+        - url: standard/library-guidance/index.md
+          itemType: concept
+          text: Open Source library guidance
```
Comment (pkulikov) [inline docs/index.yml]:
```suggestion
          text: Open source library guidance
```

File: docs/index.yml [hunk 3]
```diff
@@ -28,25 +28,138 @@ highlightedContent:
...
+    # Card
+    - title: .NET architectural guides
+      links:
+        - url: architecture/microservices/index.md
+          itemType: architecture
+          text: ".NET Microservices: Architecture for containerized .NET apps"
+        - url: architecture/modern-web-apps-azure/index.md
+          itemType: architecture
+          text: Architect Modern Web Applications with ASP.NET Core and Azure
```
Comment (pkulikov) [inline docs/index.yml]:
```suggestion
          text: Architect modern web applications with ASP.NET Core and Azure
```

File: docs/index.yml [hunk 4]
```diff
@@ -28,25 +28,138 @@ highlightedContent:
...
+          text: .NET Core CLI overview
+        - url: core/testing/index.md
+          itemType: tutorial
+          text: Unit testing with .NET Core apps
+        - url: core/porting/index.md
+          itemType: how-to-guide
+          text: Porting from .NET Framework to .NET Core
+        - url: desktop-wpf/overview/index.md
+          itemType: overview
+          text: Create .NET Core desktop apps
```
Comment (pkulikov) [inline docs/index.yml]: Should Windows be mentioned somehow? For example:
```suggestion
          text: Create .NET Core desktop apps for Windows
```

**Comment by BillWagner** [comment]
I'll merge this without other reviews due to vacation schedules.

(It's going to a branch for review, not being published yet).

**Pull request merged**